packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstantsHelper.java

```diff
@@ -50,6 +52,31 @@
         DIRECT_EVENTS_KEY, UIManagerModuleConstants.getDirectEventTypeConstants());
   }
 
+  private static void validateDirectEventNames(
+      String viewManagerName, Map<String, Object> directEvents) {
+    if (!ReactBuildConfig.DEBUG || directEvents == null) {
+      return;
+    }
+
+    for (String key : directEvents.keySet()) {
+      Object value = directEvents.get(key);
+      if (value != null && (value instanceof Map)) {
+        String regName = (String) ((Map) value).get("registrationName");
```

Comment:
Just for curiosity;
Provided L63 already checks if `value` is an `instanceof` `Map`, why do we need another explicit cast to `Map` at L64?

I think this could be simplified :)

Reply:
Ping @rshest :) seems you missed this?